### PR TITLE
Update Set-CsAutodiscoverConfiguration.md

### PR DESCRIPTION
**Pull request by priyanshamishra**
Adding clarity on the parameter : -EnableCORS

### CODE REVIEW

**Comment by priyanshamishra** [comment]
@MicrosoftDocs/marveldocs-pubops PR is ready for review. Please review and publish.

**Comment by Kateyanne** [comment]
@JohanFreelancer9 please edit. @e0i fyi

**Review by get-itips — CHANGES_REQUESTED** [review]
Please add the parameter to the SYNTAX section

**Comment by Kateyanne** [comment]
@yogkumgit fyi

File: skype/skype-ps/skype/Set-CsAutodiscoverConfiguration.md [hunk 1]
```diff
@@ -132,6 +132,19 @@ Accept pipeline input: False
...
+
+If set to True, Cross-Origin Resource Sharing is enabled.
+
+```yaml
+Type: Boolean
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+Applies to: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsAutodiscoverConfiguration.md:145]:
```suggestion
Type: Boolean
Parameter Sets: (All)
Aliases: 
Applicable: Lync Server 2013, Skype for Business Server 2015, Skype for Business Server 2019

Required: False
Position: Named
Default value: False
Accept pipeline input: False
Accept wildcard characters: False
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Just a smal detail.

**Comment by yogkumgit** [comment]
@priyanshamishra Please review and update. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**